packages/node_modules/samples/browser-socket/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Sample: Socket</title>
+    <link rel="stylesheet" href="https://unpkg.com/spectre.css/dist/spectre.min.css">
```

Comment:
Would be awesome to use collab-ui here

Reply:
Agreed. I just needed something quick and supported. I would rather not include it as another build step. I may use https://unpkg.com/@collab-ui/core/css/collab-ui.min.css when revising.